applicationinsights.json

```diff
@@ -1,12 +1,16 @@
 {
   "role": {
-    "name": "application-name"
+    "name": "calculate-journey-variable-payments"
```

Comment:
The application name was an omission on my part in the original commit of this file.